scripts/update-se-drivers.js

```diff
@@ -0,0 +1,65 @@
+const fs = require('fs');
+const cheerio = require('cheerio');
+const got = require('got');
+const chromeDriverDownloadUrl = 'https://chromedriver.chromium.org/downloads';
+const { execSync } = require('child_process');
+
+let chromeVersion;
+let chromeDriverVersion;
+let foundDriverVersion = false;
+
+function getOS() {
+  var os = process.platform;
+  if (os == "darwin") {
+    os = "MacOS";
+  } else if (os == "win32" || os == "win64") {
+    os = "Windows";
+  } else if (os == "linux") {
+    os = "Linux";
+  }
+  return os;
+}
+
+got(chromeDriverDownloadUrl).then(async(response) => {
```

Comment:
we already have `axios`. are we able to use it?

Reply:
Updated to use `axios`